cms/templates/js/course-video-settings-update-settings-footer.underscore

```diff
@@ -0,0 +1,13 @@
+<button class='button-link action-cancel-course-video-settings' aria-describedby='cancel-button-text'>
+    <%- gettext('Reset') %>
+    <span id='updcancel-button-text' class='sr'><%-gettext('Click Reset to discard changes.') %></span>
+</button>
+<button class="button action-update-course-video-settings" aria-describedby='update-button-text'>
+    <%- gettext('Update Settings') %>
+    <span id='update-button-text' class='sr'><%-gettext('Press update settings to update course video settings') %></span>
```

Comment:
Suggest "Update course video settings." If that doesn't work, I recommend "Click **Update Settings** to update course video settings." (The name of the button should be bold and use title caps.

Reply:
Actually this is screen reader text, so it won't be shown.